### ISSUE TITLE
How is the loss after transfer learning?

### ISSUE THREAD

**Issue by Docurdt** [ncoudray/DeepPATH#21]
I am trying to use about 100 slides to fine tune the Inception model to go through all the steps, but the loss dropped very very slowly. Is it normal? Should I use more slides for the training? Expecting your reply, thank you very much!
<img width="637" alt="screen shot 2018-11-14 at 22 57 50" src="https://user-images.githubusercontent.com/5456534/48481580-33e6c180-e862-11e8-954b-d0d2a4b7bfd7.png">

**Comment by ncoudray** [comment]
How was the loss at the initial steps? I guess it was much higher, no? It looks to me like it has already converged.

**Comment by Docurdt** [comment]
> How was the loss at the initial steps? I guess it was much higher, no? It looks to me like it has already converged.

<img width="617" alt="screen shot 2018-11-15 at 10 06 10" src="https://user-images.githubusercontent.com/5456534/48518548-1d6e5380-e8be-11e8-9195-c7f42bb99d42.png">
Thank you for your quickly reply!!!
The initial loss value can be seen in the above picture.
The loss value seems converged and it indeed decreased a little along with the training, but I think the loss value is still too high to end the transfer training. What is the loss value when you finished the training of the inception model with the last layer fine tune? Thanks!

**Comment by ncoudray** [comment]
I don't know what you are training for ~ in any case, you should monitor with the independent validation cohort. This will tell you if it converges, over-fit, or fails. I wouldn't rely only on the loss to take a decision.

**Comment by Docurdt** [comment]
I am training the lung cancer H/E slides with 2-class classification, normal tissue or cancer. I think the loss value should be the lower the better for the validation before the model overfit. I don't know which step I made some mistake or misunderstanding. As for the patch label, the diagnostic result (cancer or tumor) for one H/E slides will be used for all the patches from it. Is it right? Otherwise, should we manually remove some patches that only contain the normal tissues?

**Comment by ncoudray** [comment]
You should definitely run the validation -that will tell you how the AUC evolves after each iterations. It should increase and either reach a plateau, or decrease if there is over-fitting.

**Comment by Docurdt** [comment]
> You should definitely run the validation -that will tell you how the AUC evolves after each iterations. It should increase and either reach a plateau, or decrease if there is over-fitting.
Yep, thank you very much for your kindly advice.
As for the patch label, the diagnostic result (cancer or tumor) for one H/E slides will be used for all the patches from it. Is it right? Otherwise, should we manually remove some patches that only contain the normal tissues?

**Comment by ncoudray** [comment]
Correct.
You can if you want to but it would take you ages to do this.
If you really want to select regions of interest, I would advise to use Aperio beforehand on your svs images

**Comment by Docurdt** [comment]
@ncoudray Thank you very much!
